hazelcast/src/main/java/com/hazelcast/instance/impl/Node.java

```diff
@@ -911,4 +925,103 @@ public boolean isMaster(Address address) {
         Address masterAddress = clusterService.getMasterAddress();
         return server.doAddressesMatch(masterAddress, address);
     }
+
```

Comment:
Maybe move this out to its own class? You already have a separate test class for it.

Reply:
Good point, the `Node` itself is long enough already.